### DISCUSSION TITLE
unable to `uv add httpx` or `uv pip install httpx` on windows

### DISCUSSION THREAD

**Discussion by zkurtz** [encode/httpx#3417] [Potential Issue]
I always get an error like `.venv\...\Scripts\httpx.exe: Access is denied. (os error 5)`. The obvious solution is to run as admin, but in my corporate environment this would require involving IT. I wonder what could be the underlying issue. ([Cross posted](https://github.com/astral-sh/uv/issues/9476) to uv issues)

**Comment by lovelydinosaur** [comment]
Going to have to refer you to the uv team‘s response here.
